evaluate.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import os
+import argparse
+import numpy as np
+import matplotlib.pyplot as plt
+
+
+# Parse session directories
+parser = argparse.ArgumentParser(description='Plot performance of a session over training time.')
+parser.add_argument('--session_directory', dest='session_directory', action='store', type=str, help='path to session directory for which to measure performance')
+parser.add_argument('--method', dest='method', action='store', type=str, help='set to \'reactive\' (supervised learning) or \'reinforcement\' (reinforcement learning ie Q-learning)')
+parser.add_argument('--num_obj_complete', dest='num_obj_complete', action='store', type=int, help='number of objects picked before considering task complete')
+args = parser.parse_args()
+session_directory = args.session_directory
+method = args.method
+num_obj_complete = args.num_obj_complete
+
+# Parse data from session (action executed, reward values)
+# NOTE: reward_value_log just stores some value which is indicative of successful grasping, which could be a class ID (reactive) or actual reward value (from MDP, reinforcement) 
+transitions_directory = os.path.join(session_directory, 'transitions')
+executed_action_log = np.loadtxt(os.path.join(transitions_directory, 'executed-action.log.txt'), delimiter=' ')
+max_iteration = executed_action_log.shape[0]
+executed_action_log = executed_action_log[0:max_iteration,:]
+reward_value_log = np.loadtxt(os.path.join(transitions_directory, 'reward-value.log.txt'), delimiter=' ')
+reward_value_log = reward_value_log[0:max_iteration]
+clearance_log = np.loadtxt(os.path.join(transitions_directory, 'clearance.log.txt'), delimiter=' ')
```

Comment:
Did we copy over the clearance.log.txt file? What did they put in this file?

Reply:
I think all the log files would be generated during testing/training?